README.md

```diff
@@ -116,5 +116,38 @@ func main() {
         return
     }
 }
+```
 
-
+## How to Contribute
+
+All kind of contributions are welcome (e.g., code, ideas, bugs, comments, documentations, etc.)! 
```

Comment:
I think it is very good idea. Initially I wrote instructions having ```develop``` branch in mind, but then I saw that you removed, thus I adapted instructions. 
Now I am going to change the instructions and write ```develop``` instead of ```main```.

Reply:
You can find changes here: https://github.com/dmarro89/dare-db/pull/29/commits/ecaeb1a9015665b4191563128241d3fa6e9de0b6